src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.cs

```diff
@@ -146,6 +146,18 @@ protected internal override int ConvertPinNumberToLogicalNumberingScheme(int pin
         /// <inheritdoc/>
         protected internal override void Write(int pinNumber, PinValue value) => _internalDriver.Write(pinNumber, value);
 
+        /// <inheritdoc />
+        protected internal override AlternatePinMode GetAlternatePinMode(int pinNumber)

```

Comment:
@joperezr if we go ahead with GpioPin then these methods should be moved there as a property